tests/chainer_tests/training_tests/extensions_tests/test_plot_report.py

```diff
@@ -17,14 +17,16 @@ def test_available(self):
         with warnings.catch_warnings(record=True) as w:
             self.assertEqual(extensions.PlotReport.available(), available)
 
-        # It shows warning only when matplotlib.pyplot is not available
+        # It shows warning only when matplotlib is not available
         if available:
             self.assertEqual(len(w), 0)
         else:
             self.assertEqual(len(w), 1)
 
+    # In Python 2 the above test does not raise UserWarning,
```

Comment:
Sorry, I could not find out the relationship between the comment and the actual unit test.